.github/workflows/use-deploy-ecs.yml

```diff
@@ -16,6 +16,9 @@ on:
       aws-access-key-id:
         required: true
         description: "AWS_ACCESS_KEY_ID value"
+      aws-role-arn:
```

Comment:
If you're adding a required field, you need to make sure it's passed in everywhere: https://github.com/mbta/dotcom/actions/runs/6549188295

You can also probably get rid of the `aws-access-key-id` and `aws-secret-access-key` parameters.

Reply:
Thanks! I'd assumed that removing the access key and secret key will affect other steps, like [logging into ECR](https://github.com/mbta/dotcom/blob/3c2993d7d51521762a0be65c89b88315961a459a/.github/workflows/use-deploy-ecs.yml#L63C18-L63C18) and [uploading assets into S3](https://github.com/mbta/dotcom/blob/3c2993d7d51521762a0be65c89b88315961a459a/.github/workflows/use-deploy-ecs.yml#L79C2-L79C2). If that's not the case, I can certainly remove these required fields